test-client-server-genhtml

```diff
@@ -2,14 +2,29 @@
 
 from __future__ import print_function
 
-import os, sys, glob
+import os, sys, glob, argparse
 from network_testing.client_server_genhtml import build
 
-path = "./json-output/test-client-server-*.json"
-files = glob.glob(path)
+data_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'data')
 
-if not files:
-    print("File not found: {}".format(path), file=sys.stderr)
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument('-s', '--staticdatadir', action='store', default=os.path.join(data_path, 'report/static_data/'),
+            help='Alternate directory containing static data such as test descriptions')
+    parser.add_argument('-o', '--output', action='store', default=os.path.join('.', 'html-output/'),
+            help='Directory for output (will be overwritten)')
+    parser.add_argument('-t', '--templatedir', action='store', default=os.path.join(data_path, 'report/templates/'),
+            help='Directory containing HTML templates')
+    parser.add_argument('input_file', nargs='*', help='Files what will be used as input')
+    args = parser.parse_args()
 
-sys.argv = [sys.argv[0]] + files
-build()
+    path = "./json-output/test-client-server-*.json"
+    files = glob.glob(path)
+    if not files:
+        print("File not found: {}".format(path), file=sys.stderr)
+
+    input_file = args.input_file + files
+    if len(input_file) < 1:
+        print("No input JSON files to parse nothing in {} and also nothing from CMD line".format(path), file=sys.stderr)
+        sys.exit(1)
+    build(args.staticdatadir, args.output, args.templatedir, input_file)
```

Comment:
I think so, because console scripts should be installed as entry-points using Python setuptools. Using your approach, this can not be done. It is irrelevant what the code does.